AirportTicketBookingSystem/AirportTicketBookingSystem.Tests/RepositoriesTests/BookingRepositoryTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using AirportTicketBookingSystem.Adapters;
+using AirportTicketBookingSystem.DTOs;
+using AirportTicketBookingSystem.Repositories;
+using AutoFixture;
+using FluentAssertions;
+using Moq;
+
+namespace AirportTicketBookingSystem.Tests.RepositoriesTests
+{
+    public class BookingRepositoryTests
```

Comment:
FYI, repositories don't usually have much business logic to test, the tests that would matter is to test against an actual storage to make sure data are stored and retrieved correctly, and this is usually don't with integration tests. I think integration tests are not in the scope of our exercise so we keep to test against actual data. 

You can leave those tests, this is a good practice, good job!